lib/common/constant.go

```diff
@@ -26,6 +26,11 @@ const (
 
 	// BlockHeightEndOfInflation sets the block height of inflation end.
 	BlockHeightEndOfInflation uint64 = 36000000
+	// UnfreezingPeriod is the blockheight duration for unfreezing.
```

Comment:
UnfreezingPeriod defines the number of blocks required for unfreezing

Reply:
@Geod24 Yes I changed it.

Reply:
I don't see the change ?